### ISSUE TITLE
Badges button on Progress and Ranking page should be renamed to My Badges

### ISSUE THREAD

**Issue by mosefer** [NationalSecurityAgency/skills-service#1191]
Button text should match the destination "page" title, rename Badges button to My Badges to match the destination page title (which is My Badges)

**Comment by mosefer** [comment]
:+1:
